crates/nu-command/src/platform/input/input_.rs

```diff
@@ -218,9 +218,14 @@ impl Command for Input {
                 example: "let user_input = (input --default 10)",
                 result: None,
             },
+            Example {
+                description: "Get multiple lines of input from the user (newlines can be entered using `Alt` + `Enter` or `Ctrl` + `Enter`), and assign to a variable",
+                example: "let multiline_input = (input --reedline)",
+                result: None,
+            },
             Example {
                 description: "Get input from the user with history, and assign to a variable",
-                example: "let user_input = ([past,command,entries] | input )",
+                example: "let user_input = ([past,command,entries] | input)",
```

Comment:
That example actually doesn't work without `--reedline`, you need to pass that or any of the `history` flags for the history to work (which enables the reedline implementation in turn)
```suggestion
                example: "let user_input = ([past,command,entries] | input --reedline)",
```